pydatastructs/linear_data_structures/linked_lists.py

```diff
@@ -581,3 +583,133 @@ def extract(self, index):
         elif index == 0:
             self.tail.next = self.head
         return node
+
+class SkipList(object):
```

Comment:
Why it doesn't inherit `LinkedList`?

Reply:
I hope skip list having a linkedlist implementation will lead to another class of sorted linked list or sorting the linkedlist each insertion and maintaining the layer will obviously need the help of tree like structure, it's better to implement them in separate node which can act same.